### DISCUSSION TITLE
Is there a way to print output html in terminal instead file?

### DISCUSSION THREAD

**Discussion by enginecan** [bootstrap-email/bootstrap-email#228] [Help]
I would like to get a string with HTML code when I execute bootstrap-email in Python. Is there a way to achieve this?

**Comment by stuyam** [comment]
If you are using the bootstrap email gem and using it via the CLI it will default to printing a string to the terminal output. Only if you specify an output file will it save the contents to an output file: https://bootstrapemail.com/docs/usage#command-line
